json_schemas/agreement-details.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "minProperties": 1,
+  "properties": {
+    "signerName": {
```

Comment:
OK maybe not then.  I was thinking of Laurence's admin app, but I think the email we will want to surface there is who clicked the "countersign" button.